x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -234,12 +235,16 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
         version: this.context.env.packageInfo.version,
         ml: plugins.ml,
         lists: plugins.lists,
+        errorService: core.getStartServices().then(async ([coreStart]) => {
+          return coreStart.savedObjects.createInternalRepository();
+        }),
```

Comment:
nit, is it possible to use an await/async or does it have to be a then still?

Reply:
I copied this from our telemetry code - I think the idea was to make sure we are not awaiting inside of the setup function. I can take a stab at re-working this though.